src/NEPTypes.jl

```diff
@@ -1004,33 +1004,40 @@ Returns true/false if the NEP is sparse (if compute_Mder() returns sparse)
     include("nep_transformations.jl")
 
     # structure exploitation for DEP (TODO: document this)
-    function compute_Mlincomb(nep::DEP,λ::Number,V::AbstractVecOrMat,
+    function compute_Mlincomb!(nep::DEP,λ::Number,V::AbstractVecOrMat,
                               a::Vector=ones(eltype(V),size(V,2)))
         n=size(V,1); k=size(V,2);
         Av=get_Av(nep)
         # determine type froom greates type of (eltype probtype and λ)
-        T=promote_type(promote_type(eltype(V),typeof(λ)),eltype(Av[1]),eltype(nep.tauv))
-        if (k>1)
-            broadcast!(*,V,V,transpose(a))
+        TT=promote_type(promote_type(eltype(V),typeof(λ)),eltype(Av[1]),eltype(nep.tauv),eltype(a))
+
+        if (V isa AbstractVector)
+            V=a[1]*V;
```

Comment:
More efficient?  `rmul!(V,a[1])`?

Reply:
Yes! For a sparse DEP of size 5 millions we have:
Old version 
  `0.575768 seconds (37 allocations: 648.500 MiB, 42.12% gc time)`
New version
`  0.489866 seconds (32 allocations: 495.912 MiB, 35.11% gc time)`
(code comes in next commit)